github-api.php

```diff
@@ -2501,7 +2501,8 @@ function vipgoci_github_pr_review_submit(
 	$results,
 	$informational_msg,
 	$github_review_comments_max,
-	$github_review_comments_include_severity
+	$github_review_comments_include_severity,
+	int $skip_large_files_limit = VIPGOCI_VALIDATION_MAXIMUM_LINES_LIMIT
```

Comment:
Suggest not to specify a default and rely on the caller providing a value. This way the default is applied only in one place, `main.php`.

Reply:
Also, the constant is not used in two places where it should be used:

* https://github.com/Automattic/vip-go-ci/blob/hotfix/line-limit-feedback-1-0-9/main.php#L30
* https://github.com/Automattic/vip-go-ci/blob/hotfix/line-limit-feedback-1-0-9/main.php#L831

I think these are the only two places in the main code where the constant should be used, everywhere else the value configured in `$options` should be used (whether directly from `$options` or via intermediate variables).